src/acquire.c

```diff
@@ -317,13 +317,14 @@ acquire_configure(struct AcquireRuntime* self_,
                 TRACE("Configured video stream %d.", istream);
             } else {
                 TRACE("Failed to configure video stream %d.", istream);
+                self->valid_video_streams &= ~(1 << istream);
```

Comment:
Looking at this again, it doesn't look necessary to unset the bit

Reply:
Couldn't hurt, may be useful for inspecting later.